polyfills/Array/from/polyfill.js

```diff
@@ -1,14 +1,15 @@
 
 // Wrapped in IIFE to prevent leaking to global scope.
 (function () {
-	function parseIterable(arraylike) {
-		if (typeof arraylike[Symbol.iterator] !== 'function') {
+	var iterator = typeof Symbol === 'function' ? Symbol.iterator : '@@iterator';
```

Comment:
Thanks for reverting this. Polyfilling iterators is a rather tough challenge to get 100% correct.